released_plugins/v3d_plugins/MIND/soma_segmentation_plugin.pro

```diff
@@ -0,0 +1,14 @@
+QT += core gui
```

Comment:
Sorry I am confused by the change in the pro file here - there was already a pro file (`soma_segmentation.pro`) that was automatically created. Why do we need to add this one?